perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -383,6 +408,30 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Deletes given services.
+	 *
+	 * @param services List<Service> Services
```

Comment:
In openapi you expect services to be a list of ids, please either change it there or here